components/base/kinematicbase/ptgKinematics.go

```diff
@@ -167,7 +190,10 @@ func (ptgk *ptgBaseKinematics) Kinematics() referenceframe.Frame {
 func (ptgk *ptgBaseKinematics) CurrentInputs(ctx context.Context) ([]referenceframe.Input, error) {
 	ptgk.inputLock.RLock()
 	defer ptgk.inputLock.RUnlock()
-	return ptgk.currentState.currentInputs, nil
+
+	planningFrameInputs := ptgk.currentState.currentInputs
```

Comment:
I'd be ok with this returning both the localization + planning frame's inputs

Reply:
It feels like that would break the abstraction of the frames we have to amalgamate the two inputs